src/main/windows/main.js

```diff
@@ -26,10 +26,17 @@ var tray = require('../tray')
 var HEADER_HEIGHT = 38
 var TORRENT_HEIGHT = 100
 
-function init () {
+function init (state) {
   if (main.win) {
     return main.win.show()
   }
+
+  var defaultBounds = {
```

Comment:
This could even be `config.DEFAULT_BOUNDS`. If you move it there, I'd also replace the `TORRENT_HEIGHT * 6` with something like `TORRENT_HEIGHT * (DEFAULT_TORRENTS.length + 1)`